api/db/apd.js

```diff
@@ -11,8 +11,11 @@ module.exports = () => ({
     },
 
     static: {
+      updateableFields: ['status', 'period'],
+      foreignKey: 'apd_id',
+      owns: { activities: 'apdActivity' },
       withRelated: [
-        'activities',
+        { activities: query => query.orderBy('id') },
```

Comment:
This is how we can specify how to sort these related models.  I only did it on activities here, but we might want to do it on all of them later.  More on that in a minute...